stl/inc/future

```diff
@@ -173,7 +173,7 @@ class _Associated_state;
 template <class _Ty>
 struct __declspec(novtable) _Deleter_base { // abstract base class for managing deletion of state objects
     virtual void _Delete(_Associated_state<_Ty>*) noexcept = 0;
-    virtual ~_Deleter_base() noexcept {}
+    virtual ~_Deleter_base()                               = default;

```

Comment:
Great work, clang-format. 🙄 (No change requested.)

Reply:
Fortunately, clang-format can be tweaked by the long comment you suggested.😎

Reply:
Oooh, nice!